frontend/src/lib/SegmentMetricsManager.ts

```diff
@@ -188,7 +188,11 @@ export class SegmentMetricsManager implements MetricsManager {
     if (IS_DEV_ENV) {
       logAlways("[Dev mode] Not tracking stat datapoint: ", evName, data)
     } else {
-      this.track(evName, data)
+      this.track(evName, data, {
+        context: {
+          ip: "0.0.0.0",
```

Comment:
nit: Maybe can you add a comment here to make it more explicit why we are doing this. Something like:

// Segment automatically attaches the IP address. But we don't use, process, or store IP addresses for our telemetry. To make this more explicit, we are overwriting this here so that it is never even sent to Segment.

Reply:
Done